begin.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>coding101</title>
+	<header>
+		<img src="images/logo.png">
+		<p>Coding101</p>
+	</header>
+</head>
+<body>
+  <section>
```

Comment:
***Code Quality: Nitpick***
This should be revisited and divided into reasonable divs